src/mappings/index.ts

```diff
@@ -156,6 +160,14 @@ export async function handleTokenCreate(context: Context): Promise<void> {
   collection.nftCount += 1;
   collection.supply += 1;
 
+  const { ownerCount, distribution } = await calculateCollectionOwnerCountAndDistribution(
+    context.store,
+    final.id,
+    final.currentOwner,
+  );
+  collection.ownerCount = ownerCount;
+  collection.distribution = distribution;
+
```

Comment:
mint won't change much distribution imo

Reply:
can only the owner of the collection mint new NFTs?

Reply:
yup

Reply:
ahh ok, sorry about that

Reply:
Do not worry will try to make docs.  🤗